src/main/java/com/cloudbees/jenkins/plugins/docker_build_env/Docker.java

```diff
@@ -57,6 +61,14 @@ public Docker(DockerServerEndpoint dockerHost, String dockerInstallation, String
         this.build = build;
         this.verbose = verbose | debug;
         this.privileged = privileged;
+        this.sudo = sudo;
+
+        this.protectedEnvironmentVariables = new HashSet<String>();
+        if(protectedEnvironmentVariables != null) {
+            for (final String variable : Arrays.asList(protectedEnvironmentVariables.split(","))) {
+                this.protectedEnvironmentVariables.add(variable.trim());
```

Comment:
There is a risk of getting empty strings here. IMHO blank strings should be skipped in this cycle